### ISSUE TITLE
Build repo add/remove command line options

### ISSUE THREAD

**Issue by forstmeier** [heupr/heupr-cli#2]
#### Description

After the Heupr instance has been launched, there may be a need to add/remove repos while the instance is running. These commands would be initial options to control that.

#### Specifics

- [ ] `add <owner/repo>`
  - [ ] required argument owner/repo string
  - [ ] updates in-memory collection
  - [ ] references `.heupr.toml` file in repo
- [ ] `delete <owner/repo>`
  - [ ] required argument owner/repo string
  - [ ] removes repo from in-memory collection

These will be necessary after #1 has been completed (I think they will need to happen in order but I could be wrong about that); one of the additional challenges on this one will be ensuring graceful termination/addition to events currently running through the pipeline.

Examples:
- `heupr repo add heupr/heupr-cli`
- `heupr repo delete heupr/heupr-cli`

Hey, @taylormike and @forstmeier - here's a new issue for you to check out!
